runbotics/runbotics-orchestrator-ui/src/src-app/views/auth/RegisterPage/Register.page.tsx

```diff
@@ -97,6 +98,23 @@ const RegisterPage: FC = () => {
     const dispatch = useDispatch();
     const { enqueueSnackbar } = useSnackbar();
 
+    const invitingTenant = useSelector((state) => tenantsSelector(state).invitingTenant);
+
+    useEffect(() => {
+        dispatch(tenantsActions.fetchTenantNameByInviteCode(inviteCodeURL))
+            .unwrap()
+            .catch((error) => {
+                if (error.statusCode === 400) {
+                    router.replace('/404');
```

Comment:
I think you can remove this if for now and leave redirect for this 404 site (because as far as I know we don't have still specific sites for erros)